src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -220,6 +220,25 @@ public Mono<Void> createModificationGroup(UUID sourceGroupUuid, UUID groupUuid,
         return getModifications(sourceGroupUuid, false, false).doOnNext(m -> {
             Optional<ModificationEntity> modification = this.modificationRepository.findById(m.getUuid());
             modification.get().setId(null);
+            //TODO is there a better way ? forced to do the following setting id to null to not get a persist detached object exception on the currentLimits
+            if (modification.get().toModificationInfos().getType().equals(ModificationType.LINE_CREATION)) {
```

Comment:
There are other entities with relationships : 
LineAttachToVoltageLevelEntity (@OneToOne with VoltageLevelCreationEntity and LineCreationEntity)
LineSplitWithVoltageLevelEntity (@OneToOne with VoltageLevelCreationEntity)

not handled here ...

Reply:
fixed